lib/pure/asyncfutures.nim

```diff
@@ -299,20 +312,23 @@ proc `$`*(stackTraceEntries: seq[StackTraceEntry]): string =
   else:
     let entries = stackTraceEntries
 
-  result = ""
```

Comment:
Don't delete this assignment, Nim will get definite assignment analysis for all types.

Reply:
Empty string assignments brought back.